src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Service/CreateServiceCommandHandler.php

```diff
@@ -51,6 +51,12 @@ public function handle(CreateServiceCommand $command)
         $service->setTeamName($command->getTeamName());
         $service->setProductionEntitiesEnabled($command->isProductionEntitiesEnabled());
         $service->setPrivacyQuestionsEnabled($command->isPrivacyQuestionsEnabled());
+        $service->setServiceType($command->getServiceType());
```

Comment:
If all these values are set in the constructor I think it will  make the code better and less prone to errors. I don't think it's an issue now but if we're tackling technical debt we could consider to refactor this.

Reply:
:+1: